administrator/language/de-DE/com_joomlaupdate.ini

```diff
@@ -74,38 +93,39 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="Es sind keine Erweiterungen insta
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PRE_UPDATE_CHECKS_FAILED="Kompatibilitätsprüfung fehlgeschlagen"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PRE_UPDATE_CHECKS_FAILED_NOTES="Es war nicht möglich, die Kompatibilität aller Erweiterungen zu prüfen. Die Abfrage beim Update-Server hat entweder ein Zeitlimit überschritten oder einen Fehler zurückgegeben."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE="Kein Update erforderlich"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>Der Entwickler der Erweiterung gibt an, dass die derzeit installierte Version kompatibel ist.</p><p id='updateyellowwarning' class='hidden'>Bitte beachten, ist eine Version mit <span class='label label-warning'>X.X.X</span> gekennzeichnet, wird vom Entwickler für die aktuelle Version von Joomla eine neuere Version der Erweiterung angeboten, als für die neue Version von Joomla. Bitte vor dem Update den Entwickler kontaktieren.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>Der Entwickler der Erweiterung gibt an, dass die derzeit installierte Version kompatibel ist.</p><p id=\"updateyellowwarning\" class=\"hidden\">Bitte beachten, ist eine Version mit <span class='label label-warning'>X.X.X</span> gekennzeichnet, wird vom Entwickler für die aktuelle Version von Joomla eine neuere Version der Erweiterung angeboten, als für die neue Version von Joomla. Bitte vor dem Update den Entwickler kontaktieren.</p>"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE="Update erforderlich"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id='updateorangewarning' class='hidden'>Bitte besonders vorsichtig sein, wenn die zu aktualisierende Version der Erweiterung nicht auch als kompatibel mit der aktuellen Version von Joomla aufgeführt ist.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla gelistet ist!</p>"
```

Comment:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla! gelistet ist</p>"
```

Reply:
Vorsichtig mit den !

Reply:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla gelistet ist.</p>"
```

Reply:
Würde hier dennoch auf das ! hinter Joomla verzichten. Eigentlich verwenden wir das im Text ja so nicht mehr.

Reply:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla! gelistet ist.</p>"
```

Reply:
Eigentlich sollte das "!", wenn man den CI-Richtlinien folgt immer verwendet werden. Aber ich sehe das Joomla auch sehr oft ohne "!"

Reply:
In einem Body-Text ist es optional unter gewissen Voraussetzungen.
https://docs.joomla.org/Joomla:Proper_use_of_the_Joomla!_name
Die sehe ich hier gegeben.

Wir können sie an diesen beiden Stellen aber auch einfügen um das Produkt Joomla! auszudrücken.
Machen wir sie rein.